static/docs/command-reference/config.md

```diff
@@ -169,8 +175,8 @@ for more details.) This section contains the following options:
 
 ### state
 
-See [DVC Files and Directories](/doc/user-guide/dvc-files-and-directories) to learn
-more about the state file (database) that is used for optimization.
+See [DVC Files and Directories](/doc/user-guide/dvc-files-and-directories) to
```

Comment:
Same here.

Reply:
yep, it's a problem. Looks like we need to run the full pass once again and fix the master branch check to test everything (not staged only files)

Reply:
Again, this was also already fixed in `master` in f36b098 (see in [blame](https://github.com/iterative/dvc.org/blame/master/static/docs/command-reference/config.md#L172)). I don't understand why Github shows this as a difference. As to how this formatting broke, that I don't know, seems intriguing indeed.

Reply:
This one broke in 72406e0 (same author, I guess same PR too probably).

Reply:
(Same as https://github.com/iterative/dvc.org/pull/867#discussion_r358667162)